samples/drumthumper/src/main/java/com/plausibleaudio/drumthumper/DrumPlayer.kt

```diff
@@ -22,18 +22,19 @@ import java.io.IOException
 class DrumPlayer {
     companion object {
         // Sample attributes
-        val NUM_CHANNELS: Int = 2       // The number of channels in the player Stream.
+        val NUM_PLAY_CHANNELS: Int = 2  // The number of channels in the player Stream.
                                         // Stereo Playback, set to 1 for Mono playback
                                         // This IS NOT the channel format of the source samples
                                         // (which must be mono).
+        val NUM_SAMPLE_CHANNELS: Int = 1;   // All WAV resource must be mono
         val SAMPLE_RATE: Int = 44100    // All the input samples are assumed to BE 44.1K
                                         // All the input samples are assumed to be mono.
 
         // Sample Buffer IDs
         val BASSDRUM: Int = 0
         val SNAREDRUM: Int = 1
         val CRASHCYMBAL: Int = 2
-        val RIDECYMBAL: Int = 3
+        const val RIDECYMBAL: Int = 3
```

Comment:
Why is just this one val declared const?

Reply:
Hmmm..... "const" isn't even a Kotlin thing.